### PR TITLE
Make network test backward compatible with 3.4 and earlier versions

### PR DESCRIPTION
**Pull request by schituku**
Added a new -v/--version option to the test which will decide the appropriate options for the versions

### CODE REVIEW

**Comment by schituku** [comment]
fixes #153

File: networking/synthetic/config.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 environment:
-  master: master.example.com
-  nodes: node1.example.com node2.example.com
\ No newline at end of file
+  master: ip-172-31-22-251.us-west-2.compute.internal
+  nodes: ip-172-31-13-12.us-west-2.compute.internal ip-172-31-33-87.us-west-2.compute.internal
+  version: 3.5
```
Comment (mffiedler) [inline networking/synthetic/config.yaml]: Please remove specific AWS addresses

File: networking/synthetic/network-test.py [hunk 2]
```diff
@@ -71,6 +72,12 @@ def parse_args():
     parser.add_argument('test_type',
                         choices = ['podIP', 'svcIP'])
     
+    parser.add_argument('-v',
+                        '--version',
+                        required = True,
```
Comment (mffiedler) [inline networking/synthetic/network-test.py]: Might be nicer to default to 3.5 mode

File: networking/synthetic/config.yaml (hunk 1 above)
Comment (schituku) [inline networking/synthetic/config.yaml]: @mffiedler  I realized this after the commit so corrected in the second commit with the comment - "new config file"

File: networking/synthetic/network-test.py (hunk 2 above)
Comment (schituku) [inline networking/synthetic/network-test.py]: @mffiedler made the 3.5 as default

**Comment by schituku** [comment]
@mffiedler closed the review comments and tested it. I think it is good to go for a merge.

**Pull request merged**